gpflow/models/svgp.py

```diff
@@ -170,3 +175,75 @@ def predict_f(self, Xnew: InputData, full_cov=False, full_output_cov=False) -> M
         )
         # tf.debugging.assert_positive(var)  # We really should make the tests pass with this here
         return mu + self.mean_function(Xnew), var
+
+
+class NewSVGP(OldSVGP):
+    """
+    Adds posterior() method and uses different math ordering for predict_f
+    """
+
+    def __init__(
```

Comment:
It doesn't look like you are changing anything in the init compared to the super class, so you could just drop the init here.